pkg/apis/v1beta1/ec2nodeclass.go

```diff
@@ -316,6 +319,24 @@ type BlockDevice struct {
 	VolumeType *string `json:"volumeType,omitempty"`
 }
 
+// InstanceStoreConfigurationOption enumerates options for configuring instance store disks.
+// See: https://github.com/awslabs/amazon-eks-ami/blob/master/doc/USER_GUIDE.md
+// +kubebuilder:validation:Enum=MountNodeEphemeralStorage;MountIndividualDisks
+type InstanceStoreConfigurationOption string
```

Comment:
```suggestion
type InstanceStorePolicyOption string
```

I wonder if this is really more of a policy in upstream terms rather than a configuration.

Reply:
Policy makes sense to me, changed in https://github.com/aws/karpenter/commit/5cc1de801310e130bf6332da9ac71d94f4dd9228